go.mod

```
@@ -0,0 +1,29 @@
+module github.com/gravitational/license
+
+go 1.21
```

```suggestion
go 1.19
```

Teleport 12 is still specifying Go 1.19 as a minimum, and we don't use any newer syntax or language features in Go 1.21 so there's no need to break support.